src/content/ja/2020/javascript.md

```diff
@@ -0,0 +1,781 @@
+---
+part_number: I
+chapter_number: 2
+title: JavaScript
+description: 2020年Web Almanac「JavaScript」の章では、ウェブ上でどれだけJavaScriptを使用しているか、圧縮、ライブラリとフレームワーク、ロード、ソースマップを網羅しています。
+authors: [tkadlec]
+reviewers: [ibnesayeed, denar90]
+analysts: [rviscomi, paulcalvano]
+editors: [rviscomi]
+translators: [ksakae]
+tkadlec_bio: Timは、誰もが使えるWebを構築することに焦点を当てたWebパフォーマンスコンサルタントであり、トレーナーです。著書に『High Performance Images』（O'Reilly, 2016）、『Implementing Responsive Design』（New Riders, 2012）があります。彼は<a href="https://timkadlec.com/">timkadlec.com</a>でウェブ全般について書いています。<a href="https://twitter.com/tkadlec">@tkadlec</a>で、彼の考えを簡潔にまとめてTwitterで紹介しています。
+discuss: 2038
+results: https://docs.google.com/spreadsheets/d/1cgXJrFH02SHPKDGD0AelaXAdB3UI7PIb5dlS0dxVtfY/
+queries: 02_JavaScript
```

Comment:
Jist changed this in #1936 (note you may need to merge main into your branch to avoid this breaking the build):

```suggestion
#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
title: JavaScript
description: 2020年Web Almanac「JavaScript」の章では、ウェブ上でどれだけJavaScriptを使用しているか、圧縮、ライブラリとフレームワーク、ロード、ソースマップを網羅しています。
authors: [tkadlec]
reviewers: [ibnesayeed, denar90]
analysts: [rviscomi, paulcalvano]
editors: [rviscomi]
translators: [ksakae]
tkadlec_bio: Timは、誰もが使えるWebを構築することに焦点を当てたWebパフォーマンスコンサルタントであり、トレーナーです。著書に『High Performance Images』（O'Reilly, 2016）、『Implementing Responsive Design』（New Riders, 2012）があります。彼は<a href="https://timkadlec.com/">timkadlec.com</a>でウェブ全般について書いています。<a href="https://twitter.com/tkadlec">@tkadlec</a>で、彼の考えを簡潔にまとめてTwitterで紹介しています。
discuss: 2038
results: https://docs.google.com/spreadsheets/d/1cgXJrFH02SHPKDGD0AelaXAdB3UI7PIb5dlS0dxVtfY/
```

Reply:
Thanks, I fix.